code/index.html

```diff
@@ -3,28 +3,232 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>HIBR – It's like Uber, but for sleep.</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lexend+Peta&family=Roboto+Mono:wght@400;700&family=Roboto+Slab:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <!-- Found a helpful guide which advocated it's ok to put the JS in the head, as long as you add the "defer" property: https://gomakethings.com/when-should-you-add-the-defer-attribute-to-the-script-element/#:~:text=In%20short%3A%20with%20defer%20on,downloaded%20and%20ready%20to%20run. -->
+    <script src="script.js" defer></script>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <nav>
+      <div class="nav-content">
+        <div class="nav-logo"><img src="assets/hibr-logo.svg"></div>
```

Comment:
Super small thing - since there is an image nested inside `div`, it should be formatted in next, separate line